lib/include/jxl/codestream_header.h

```diff
@@ -304,6 +316,10 @@ typedef struct {
    */
   uint32_t name_length;
 
+  /** Color transform the frame uses.
+   */
+  JxlColorTransform color_transform;
```

Comment:
This is an API/ABI change and should be documented on the CHANGELOG.md (ideally we should make these on a different PR).

That said I'm not sure this fits our decoder library. This library is for decoding JPEG XL, not a JPEG XL parser or tokenizer library. The goal is to give you decoded pixels or lossless reconstructed JPEGs in all the useful forms you may need for efficiently displaying them. Is this field needed by the user? I'd propose you leave this change out of the PR.